Demo/src/main/java/com/braintreepayments/demo/VenmoFragment.java

```diff
@@ -43,11 +48,13 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
     }
 
     private void handleVenmoResult(VenmoAccountNonce venmoAccountNonce) {
-        super.onPaymentMethodNonceCreated(venmoAccountNonce);
+        if (venmoAccountNonce != null) {
```

Comment:
Is this just fixing lint? If this is only called in the onSuccess listener method it shouldn't ever be null I think

Reply:
Ah, good catch, removed: https://github.com/braintree/braintree_android/pull/911/commits/1bd1e1d979c7d9673797c23ffe77aece150bdb8b